tsconfig.json

```diff
@@ -39,13 +39,16 @@
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
     /* Module Resolution Options */
-    // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
     // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
-    // "typeRoots": [],                       /* List of folders to include type definitions from. */
-    // "types": [],                           /* Type declaration files to be included in compilation. */
-    // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
+    "typeRoots": [
+      "./node_modules/@types",
+      "./node_modules/@openeth",
```

Comment:
I was under the impression that `@types/*` types were automatically included.

Reply:
https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types

> By default all visible “@types” packages are included in your compilation. [...] If typeRoots is specified, only packages under typeRoots will be included.

So I have to add the "./node_modules/@types" if I want to add other paths as well.